test.go

```diff
@@ -74,7 +74,7 @@ For example:
 // Flags
 var (
 	flavor           = flag.String("flavor", "mysql57", "comma-separated bootstrap flavor(s) to run against (when using Docker mode). Available flavors: all,"+flavors)
-	bootstrapVersion = flag.String("bootstrap-version", "6", "the version identifier to use for the docker images")
+	bootstrapVersion = flag.String("bootstrap-version", "7", "the version identifier to use for the docker images")
```

Comment:
@vmg, the bootstrap version is bumped in `test.go`

Reply:
Yes, but that happened after @deepthi pushed the docker images, so they need to be pushed _again_ (this is my understanding from reading the Slack logs).

Reply:
Makes sense. we'll do this today.

Reply:
I have push a commit with all the files updated to 8 .. @deepthi

Reply:
Pushed new bootstrap images.